src-built-in/components/UIAPI/hooks/workspaces.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState, WorkspaceState } from '../types';
+import * as Actions from '../actions/workspaceActions';
+
+/**
+ * A hook for getting the activeWorkspaceName,
+ * and setting the activeWorkspaceName.
```

Comment:
You can't set the active workspace name this way.

Reply:
removed